xml/cap_depl_azure.xml

```diff
@@ -690,7 +693,7 @@ xip.io:2793/oauth/token"</screen>
 </screen>
 
 <para>
-Set up the service broker with your variables:
+Set up the service broker with your variables, then create it:
 </para>
 <screen>
 &prompt.user;helm install azure/open-service-broker-azure \
```

Comment:
#290 mentioned adding `--set tls.enabled=false` so that http url works because of the https url doesn't work with a self signed cert. Are we omitting that because we assume a self signed cert is not used?

Reply:
I wish it were on purpose for a good reason. but no :)